### ISSUE TITLE
Regeneration of spec webpages

### ISSUE THREAD

**Issue by khelm** [incf-nidash/nidm-specs#456]
I'm having issues with the code used to regenerate the specs webpages.  I'm following the instructions on the wiki page and I've installed all of the necessary packages successfully. The issue arises here:

```
$ fab serve
[localhost] local: rm -rf output
[localhost] local: mkdir output
[localhost] local: pelican content/ -s pelicanconf.py
WARNING: Feeds generated without SITEURL set properly may not be valid
ERROR: Skipping specs/README.md: could not find information about 'title'
CRITICAL: UndefinedError: '_' is undefined

Fatal error: local() encountered an error (return code 1) while executing 'pelican content/ -s pelicanconf.py'

Aborting.
```

I don't see any stray or seemingly out of place underscores or something wrong right before the occurrence of an underscore in any of the md files nor in the pelicanconf.py file.  Any ideas are welcome.

**Comment by cmaumet** [comment]
@khelm, I think our discussions at https://github.com/incf-nidash/nidm-specs/issues/453 will also be useful for this issue. If that's okay with you, let's discuss over there and then we can get back here?

**Comment by cmaumet** [comment]
(Note: I have updated your comment above to add ``` above and below your log so that it appears as a bloc of code. More info at: https://help.github.com/articles/basic-writing-and-formatting-syntax/#quoting-code)

**Comment by cmaumet** [comment]
Also open at #461

**Comment by khelm** [comment]
This has been resolved at [#471](https://github.com/incf-nidash/nidm-specs/pull/471) and will now be closed.
